.rubocop_todo.yml

```diff
@@ -26,11 +26,16 @@ RSpec/ExampleLength:
     - 'spec/requests/api/v0/articles_spec.rb'
     - 'spec/requests/display_ad_events_spec.rb'
 
-# Offense count: 427
+# Offense count: 444
 # Configuration parameters: AggregateFailuresByDefault.
 RSpec/MultipleExpectations:
   Max: 8
 
+# Offense count: 1
+RSpec/UnspecifiedException:
+  Exclude:
+    - 'spec/requests/messages_spec.rb'
```

Comment:
this is related to this PR attempt, https://github.com/thepracticaldev/dev.to/pull/4573

until we fix the root of the problem, i'm +1 on keeping this in the todo